tests/fixtures/config/interpreter_test/env/project.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+engines:
+  test_engine_with_frameworks:
+    location:
+      type: path
+      path: ${BUNDLES_ROOT}/test_engine_with_frameworks
```

Comment:
Why can't you use  `$SHOTGUN_TEST_ENGINE`?

Reply:
I didn't notice that we had a test_engine in tk-toolchain. I switched over to using it. I had my frameworks associated with the engine but it was easy enough to move them over to the app.